python/ray/tests/test_channel.py

```diff
@@ -903,15 +902,12 @@ def create_actor(node):
     a = create_actor(actor_node)
     assert driver_node != ray.get(a.get_node_id.remote())
 
-    with pytest.raises(
-        ValueError,
-        match=re.escape(
-            "The reader and writer are on different nodes, so the object written to "
-            "the channel must have a size less than or equal to the max gRPC payload "
-            "size (471859200 bytes)."
-        ),
-    ):
-        ray_channel.Channel(None, [a], 1024 * 1024 * 512)
+    size = 1024 * 1024 * 512
```

Comment:
Should we increase the `size` a bit? In `context.py`, you mentioned that Ray sets the limit to approximately 512 MiB. Maybe use 600 MiB instead? In addition, would you mind adding some comments to describe why do we decide the value of `size` in case most people don't know the size limit of gRPC payload in Ray.

Reply:
Done.